packages/nimbus/src/components/text-input/text-input.mdx

```diff
@@ -13,106 +13,180 @@ tags:
 ---
 
 # TextInput
+---
+
+Allows users to enter information like names, addresses, emails, passwords, search queries, or any other text-based data.
+
+## Overview
+
+Users can type, delete, copy, and paste text within the box. Sometimes, there can be additional features like autocomplete or validation to help users enter information correctly.
 
-The TextInput component is a form control that allows users to enter text into
-an input field. It provides a clean, accessible interface for text input with
-support for different states and variants.
+### Resources
 
-## Features
+Deep dive on details and access design library.
 
-- **Accessible:** Built with ARIA attributes for screen reader support
-- **Customizable:** Supports different sizes and visual variants
-- **State Management:** Handles disabled, invalid, and focus states
+[TECH RELATED LINK](#)
+[TECH RELATED LINK](#)
+[Figma library](https://www.figma.com/design/gHbAJGfcrCv7f2bgzUQgHq/NIMBUS-Guidelines?node-id=1696-48973&m=dev)
 
-## Basic Usage
+## Variables
+Get familiar with the features. 
 
-The TextInput component is used to create form input fields where users can enter text. It supports various states and can be customized using props.
+### Size
+
+**Medium**
+
+This is the default size for text inputs. 
 
 ```jsx-live
-const App = () => <TextInput placeholder="basic text input"/>
+const App = () => <TextInput size="md" value="input value"/>
 ```
 
-### Sizes
+**Small**
 
-Available sizes.
+Available for the times a more condensed text input is needed.
 
 ```jsx-live
-const App = () => {
-
-  const sizes = ['md', 'sm',];
-
-  return (
-    <Stack direction="horizontal" alignItems="center">
-      {sizes.map(size => (
-        <TextInput key={size} size={size} placeholder={size}/>
-      ))}
-    </Stack>
-  )
-}
+const App = () => <TextInput size="sm" value="input value"/>
 ```
 
-### Variants
+### Tone
 
-There are two variants currently:
-- **Solid:** The default variant with a solid background.
-- **Ghost:** A minimal variant without background.
+**Placeholder**
 
+This is the styling for placeholder text. It is a shade lighter than active text.
 
 ```jsx-live
-const App = () => {
-
-  const variants = ['solid','ghost',];
-
-  return (
-    <Stack direction="horizontal">
-      {variants.map(variant => (
-        <TextInput key={variant} variant={variant} placeholder={variant} />
-      ))}
-    </Stack>
-  )
-}
+const App = () => <TextInput size="sm" placeholder="input value"/>
 ```
 
-## States
+**Primary**
 
-### Disabled
+This is the default tone for inputs a user adds or is displayed within a text input.
 
-When interaction is not allowed.
+```jsx-live
+const App = () => <TextInput size="sm" tone="primary" value="input value"/>
```

Comment:
```suggestion
const App = () => <TextInput size="md" tone="primary" value="input value"/>
```

Reply:
Oh goodness! Thank you, I will just take them out since the default variant is "md"